[2024-09-02T16:10:16.644+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-09-02T16:10:16.669+0000] {taskinstance.py:2603} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: our_first_dag_v2.first_task manual__2024-09-02T16:10:15.213973+00:00 [queued]>
[2024-09-02T16:10:16.679+0000] {taskinstance.py:2603} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: our_first_dag_v2.first_task manual__2024-09-02T16:10:15.213973+00:00 [queued]>
[2024-09-02T16:10:16.680+0000] {taskinstance.py:2856} INFO - Starting attempt 1 of 6
[2024-09-02T16:10:16.694+0000] {taskinstance.py:2879} INFO - Executing <Task(BashOperator): first_task> on 2024-09-02 16:10:15.213973+00:00
[2024-09-02T16:10:16.713+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=3823) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-09-02T16:10:16.712+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'our_first_dag_v2', 'first_task', 'manual__2024-09-02T16:10:15.213973+00:00', '--job-id', '4', '--raw', '--subdir', 'DAGS_FOLDER/our_first_dag.py', '--cfg-path', '/tmp/tmpocfjkp7n']
[2024-09-02T16:10:16.715+0000] {standard_task_runner.py:72} INFO - Started process 3824 to run task
[2024-09-02T16:10:16.716+0000] {standard_task_runner.py:105} INFO - Job 4: Subtask first_task
[2024-09-02T16:10:16.762+0000] {task_command.py:467} INFO - Running <TaskInstance: our_first_dag_v2.first_task manual__2024-09-02T16:10:15.213973+00:00 [running]> on host cedbab6b7256
[2024-09-02T16:10:16.836+0000] {taskinstance.py:3122} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='abhay' AIRFLOW_CTX_DAG_ID='our_first_dag_v2' AIRFLOW_CTX_TASK_ID='first_task' AIRFLOW_CTX_EXECUTION_DATE='2024-09-02T16:10:15.213973+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-09-02T16:10:15.213973+00:00'
[2024-09-02T16:10:16.839+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-09-02T16:10:16.873+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2024-09-02T16:10:16.877+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'echo hello world, this is the first task!']
[2024-09-02T16:10:16.888+0000] {subprocess.py:86} INFO - Output:
[2024-09-02T16:10:16.894+0000] {subprocess.py:93} INFO - hello world, this is the first task!
[2024-09-02T16:10:16.895+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2024-09-02T16:10:16.914+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-09-02T16:10:16.915+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=our_first_dag_v2, task_id=first_task, run_id=manual__2024-09-02T16:10:15.213973+00:00, execution_date=20240902T161015, start_date=20240902T161016, end_date=20240902T161016
[2024-09-02T16:10:16.936+0000] {local_task_job_runner.py:261} INFO - Task exited with return code 0
[2024-09-02T16:10:16.954+0000] {taskinstance.py:3891} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-09-02T16:10:16.955+0000] {local_task_job_runner.py:240} INFO - ::endgroup::
